R/model.R

```diff
@@ -419,7 +422,8 @@ CmdStanModel$set("public", name = "compile", value = compile_method)
 #'   * `validate_csv`: (logical) When `TRUE` (the default), validate the
 #'   sampling results in the csv files. Disable if you wish to manually read in
 #'   the sampling results and validate them.
-#'
+#'   * `threads_per_chain`: (positive integer) The number of threads to use per MCMC chaing
```

Comment:
typo: "MCMC chaing" should be "MCMC chain"

Reply:
Fixed